service/utils/utils.go

```diff
@@ -17,16 +17,17 @@
 package utils
 
 import (
-	"github.com/vmware/harbor/utils/log"
 	"net/http"
 	"os"
+
+	"github.com/vmware/harbor/utils/log"
 )
 
 // VerifySecret verifies the UI_SECRET cookie in a http request.
 func VerifySecret(r *http.Request) bool {
 	secret := os.Getenv("UI_SECRET")
 	c, err := r.Cookie("uisecret")
-	if err != nil {
+	if err != nil && err != http.ErrNoCookie {
```

Comment:
Could you please do me a favor to update the log to warning instead of error? :)